frontend/src/utilities/useRefreshInterval.ts

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const useRefreshInterval = (refreshInterval, callback: () => void) => {
```

Comment:
`refreshInterval` type should be `number`?

Reply:
That gives a type error, it seems to think it's a NodeJS.Timer which doesn't return a number - this was how I got round it. I wonder if there's a better way of doing it to force it to use the browser timer instance, maybe calling window.setInterval? WDYT?